src/plot/svgMask.tsx

```diff
@@ -0,0 +1,139 @@
+import { SvgRect } from '@h5web/lib';
+import type { SVGProps } from 'react';
+import { Vector3 } from 'three';
+
+type Rect = [Vector3, Vector3];
+
+interface Props extends SVGProps<SVGPathElement> {
```

Comment:
could: I am slightly opinionated to the convention where this interface is called `SvgMaskProps`, in a case it needs to be used elsewhere later on.